# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/xdm/hotfix-glibc236.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- xdm-1.1.12/xdm/genauth.c.vanilla	2022-08-03 12:52:26.136940744 +0200
+++ xdm-1.1.12/xdm/genauth.c	2022-08-03 12:52:38.312940258 +0200
@@ -42,7 +42,7 @@
 
 #ifdef HAVE_ARC4RANDOM
 # ifdef __linux__
-#  include <bsd/stdlib.h>
+#  include <stdlib.h>
 # else
 #  include <stdlib.h>
 # endif
